pandas/core/common.py

```diff
@@ -604,3 +604,10 @@ def is_builtin_func(arg):
     otherwise return the arg
     """
     return _builtin_table.get(arg, arg)
+
+
+def fill_missing_names(names):
```

Comment:
can you type in and out here

Reply:
Done, and added version and more docstring.